level1/provider.tf

```diff
@@ -6,8 +6,8 @@ terraform {
     }
   }
   backend "s3" {
-    bucket         = "terraform-remote-state-mk-0808"
-    key            = "/remote/terraform.tfstate"
+    bucket         = "terraform-remote-state-mk-14081"
```

Comment:
I suggest that you don't delete your state bucket every time. It cost a fraction.
If you keep it - there won't be a need to update your provider config each time.